deisctl/units/deis-builder.service

```diff
@@ -7,6 +7,7 @@ TimeoutStartSec=30m
 ExecStartPre=/bin/sh -c "docker inspect deis-builder-data >/dev/null 2>&1 || docker run --name deis-builder-data -v /var/lib/docker ubuntu-debootstrap:14.04 /bin/true"
 ExecStartPre=/bin/sh -c "IMAGE=`/run/deis/bin/get_image /deis/builder` && docker history $IMAGE >/dev/null || docker pull $IMAGE"
 ExecStartPre=/bin/sh -c "docker inspect deis-builder >/dev/null && docker rm -f deis-builder || true"
+ExecStartPre=-/bin/sh -c "/sbin/losetup -f"
```

Comment:
Yes, what @carmstrong said, sorry I didn't explain it well. Inside the container, `losetup -f` returns an error rather than just-in-time creating the `/dev/loopN` devices. On the host, `losetup` works fine, and then those devices are available inside the deis-builder container.